lib/createTopLevelExpect.js

```diff
@@ -1928,6 +1938,7 @@ function createTopLevelExpect({
   return extend(expect, {
     _topLevelExpect: expect,
     _outputFormat: format,
+    _frozen: false,
```

Comment:
Where is this used?

Reply:
It's needed because `expectPrototype.child` uses `createTopLevelExpect` to create the child expect. If we omit `_frozen` as an own property here, the child's `_frozen` property will be looked up in the parent per the new prototype chain.

Removing it makes these two test cases fail:

```js
  1) freeze
       with .child()
         allows addAssertion:
     Error: Cannot add an assertion to a frozen instance, please run .clone() first
      at Function.expectPrototype.addAssertion (lib/createTopLevelExpect.js:630:11)
      at Context.<anonymous> (test/api/freeze.spec.js:113:10)
      at processImmediate (internal/timers.js:456:21)

  2) freeze
       with .child()
         allows .addStyle(...):
     Error: Cannot add a style to a frozen instance, please run .clone() first
      at Function.expectPrototype.addStyle (lib/createTopLevelExpect.js:937:11)
      at Context.<anonymous> (test/api/freeze.spec.js:153:10)
      at processImmediate (internal/timers.js:456:21)
```